packaging/files/linux/deb-post-install.sh.tmpl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+#   On Debian,
+#       $1=configure : is set to 'configure' and if $2 is set, it is an upgrade
+
+if [ -z "$2" ]; then
+  chown -R apm-server:apm-server /var/lib/apm-server /var/log/apm-server /etc/apm-server/apm-server.yml
```

Comment:
why not do this always?

Reply:
was thinking in case users have already customized this?  I don't feel that strongly about it but was trying to be cautious